### ISSUE TITLE
Deprecate foodcritic quality metric QM009 in favor of QM008

### ISSUE THREAD

**Issue by sam1el** [chef-cookbooks/cookbook-quality-metrics-rfc#66]
Supermarket is still using foodcritic as a metric which is no longer best practice. 

# Describe the problem
Foodcritic is essentially fully deprecated. We should be using cookstyle which is the best practice we give customers. 
I also feel this would help us to identify severely broken and deprecated cookbooks. 

## Software Version
This is specific to public supermarket

## Possible Solution
Replace QM009 with QM008

**Comment by tas50** [comment]
Yea @robbkidd and I would both love to switch over to Cookstyle for the quality metrics, but that's a non-trivial amount of work.

**Comment by sam1el** [comment]
yeah, I was just looking at the supermarket code. looks pretty heavy. I may dig at it a bit and see what I can come up with.

**Comment by cgunasree08** [comment]
This issue or pull request is being automatically closed because the repository is being archived as part of the [Repository Standardization Initiative](https://github.com/chef-boneyard/oss-repo-standardization-2025).
